src/udev/udev-builtin-net_id.c

```diff
@@ -1155,6 +1155,23 @@ static int names_mac(sd_device *dev, const char *prefix, EventMode mode) {
                                               "Not generating MAC name for device with MAC address of length %zu.",
                                               hw_addr.length);
 
+        const char *s_hw_addr_passthrough;
+        r = sd_device_get_property_value(dev, "ID_NET_PASSTHROUGH_MAC", &s_hw_addr_passthrough);
```

Comment:
Why not to read `auxmac` of `platform/thinkpad_acpi` directly?

Also, I think it is better to add a naming policy for this logic.

Reply:
> Why not to read auxmac of platform/thinkpad_acpi directly?

Well, I think driver specific stuff really should be done in udev rules/hwdb, not in C code. That's why I did the importing of the pass-through MAC address from the driver in udev rules, but then the comparison in C code.